x-pack/plugins/case/server/routes/api/cases/find_cases.test.ts

```diff
@@ -37,4 +38,37 @@ describe('GET all cases', () => {
     expect(response.status).toEqual(200);
     expect(response.payload.cases).toHaveLength(4);
   });
+  it(`adds 'none' connector id to cases without when 3rd party unconfigured`, async () => {
+    const request = httpServerMock.createKibanaRequest({
+      path: `${CASES_URL}/_find`,
+      method: 'get',
+    });
+
+    const theContext = createRouteContext(
+      createMockSavedObjectsRepository({
+        caseSavedObject: [mockCaseNoConnectorId],
+      })
+    );
+
+    const response = await routeHandler(theContext, request, kibanaResponseFactory);
+    expect(response.status).toEqual(200);
+    expect(response.payload.cases[0].connector_id).toEqual('none');
+  });
+  it(`adds default connector id to cases without when 3rd party configured`, async () => {
```

Comment:
nit -> we can get one more test to male sure we get the connector_id if there is one.

Reply:
I think I will apply to all the tests that applied if you agree

Reply:
I think the test should female sure we get the connector_id because the future is female